<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/preferences-desktop-color.png</normaloff>:/icons/preferences-desktop-color.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabChoose">
       <attribute name="title">
        <string>Choose server</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>143</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="pixmap">
           <pixmap>:/icons/preferences-system-network.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Choose Stella server and port</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Device:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="lineIP">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Stella Server IP (Version 4 or 6)</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="btnClearHistory">
             <property name="toolTip">
              <string>Clear history</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/edit-clear.png</normaloff>:/icons/edit-clear.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Device Port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinPortOut">
             <property name="toolTip">
              <string>Stella UDP Port</string>
             </property>
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Incoming port:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinPortIn">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="listPlugins">
             <property name="toolTip">
              <string>Plugins</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QPushButton" name="btnConnect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Function:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>143</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionShow_tray_icon"/>
    <addaction name="actionMinimize_to_tray"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuStella_host">
    <property name="title">
     <string notr="true">[stella host]</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStella_host"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/preferences-desktop-color.png</normaloff>:/icons/preferences-desktop-color.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionMinimize_to_tray">
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
  <action name="actionShow_tray_icon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show tray icon</string>
   </property>
  </action>
  <action name="actionBroadcastChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Stella Broadcast Changes</string>
   </property>
   <property name="toolTip">
    <string>Broadcast changes 1s after the last change</string>
   </property>
  </action>
  <action name="actionAboutConnection">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About Stella Server</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>Close tab</string>
   </property>
  </action>
  <action name="actionSeparator"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
